test/shopify_app/controller_concerns/token_exchange_test.rb

```diff
@@ -178,13 +183,48 @@ class TokenExchangeControllerTest < ActionController::TestCase
     end
   end
 
+  [
+    ShopifyAPI::Errors::InvalidJwtTokenError,
+    ShopifyAPI::Errors::CookieNotFoundError,
```

Comment:
Maybe we shouldn't respond to invalid cookies with a bounce page now that we'll be using the method that exclusively checks `shopify_id_token`.